docs/rule/no-invalid-aria-attributes.md

```diff
@@ -0,0 +1,22 @@
+# no-invalid-aria-attributes
+
+This rule checks for the use of invalid ARIA attributes, or any aria-\* property on an element that is not included in the [WAI-ARIA States and Properties spec](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties).
```

Comment:
Can you explain what happens when someone uses a non-existent aria attribute? E.g. `A non-existent ARIA attribute has no effect.`